docs/customizing.md

```diff
@@ -245,3 +245,277 @@ Example `pulp_install_plugins` (with bogus version values):
       pulp-rpm:
          version: "7.8.9"
 ```
+
+Deployment Scenarios
+--------------------
+
+In this section, you can find information and Ansible playbook examples that demonstrate how to
+configure the following installation scenarios:
+
+* [A single “all-in-one” Pulp server that includes the database, redis, and webserver.](#all-in-one-pulp-server-example)
+* [Pulp on one server, while using existing servers for the database, redis, and webserver.](#pulp-server-with-existing-infrastructure)
+* [Pulp on one server, with the database, redis & webserver on separate servers.](#pulp-with-separate-servers-for-services)
+* [Each and every Pulp service on a separate server.](#separate-servers-for-each-and-every-service)
+
+Depending on your requirements for high availability and better scalability, you might like to
+customize your installations according to these examples.
+
+
+#### 'All in One Pulp Server' Example
+
+This deployment consists of installing a single server that contains all Pulp services, including
+the database, redis & webserver.
+
+```
+---
+- hosts: example-pulp-server
+  vars:
+    pulp_default_admin_password: << YOUR PASSWORD FOR THE PULP APPLICATION HERE >>
+    pulp_settings:
+      secret_key: << YOUR SECRET HERE >>
+      content_origin: "http://"
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+  roles:
+    - pulp_all_services
+  environment:
+    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+```    
+
+#### Pulp Server with Existing Infrastructure
+
+This deployment consists of a single Pulp server that relies on an existing database, redis
+ and webservers.
```

Comment:
It is basically no longer feasible (since this was written 1 year ago) to use an existing webserver. Or it is, but is very difficult for users to do, and beyond the scope of this document.

So let's remove "and webservers".